example/relational/README.md

```diff
@@ -162,13 +162,15 @@ SortKey is
   ⟿ keyword:theory
 */
 type Keyword struct {
-	HashKey dynamo.IRI `dynamodbav:"prefix,omitempty"`
+	HashKey dynamo.IRI `dynamodbav:"prefix,omitempty"` => PartitionKey?
 	SortKey dynamo.IRI `dynamodbav:"suffix,omitempty"`
 }
 ```
 
 There are only a few limited ways to query data efficiently from DynamoDB. The composite sort key together with `dynamo.IRI` (`curie.IRI`) data type let application retrieve hierarchy of related items using range queries with expressions `begins_with`, `between`, `>`, `<`, etc. The `dynamo` library automates construction of these expressions.
 
+=> what would be the impact on write in this case?
```

Comment:
Application needs to explicitly know what to write and table capacity needs to support the write volume.